### ISSUE TITLE
Feature request:  support for the real `Hide Label`

### ISSUE THREAD

**Issue by dnknn** [geovanneb/envmarker#12]
<img align=right width=40% src=https://user-images.githubusercontent.com/10995302/108376238-c0e90400-723d-11eb-84a7-3645cd5dce57.png>

- This is temporary, it will redisplay after `F5` reload the page!

This is the biggest shortcoming of this extension in my opinion!

---
- ## ❥ **Expected behavior:** 
　　　✚. support for the real/Switch type `Hide Label`

- ❓ **Why need do this ? :** Can you think about it?
　　　Even if you want to keep the rules, but don't want to enable them (Currently you can only delete the rule to `OFF`), 
In other words, it can be turned `OFF` when not needed, and it can be turned `ON` quickly when needed!

- ## 💡 **How to achieve:** 
　　　Changed by default, or Add an ⬜OFF/☑ON option for this.
